search/search_repository.go

```diff
@@ -0,0 +1,151 @@
+package search
+
+import (
+	"golang.org/x/net/context"
+
+	"log"
+	"strconv"
+
+	"strings"
+
+	"github.com/almighty/almighty-core/app"
+	"github.com/almighty/almighty-core/models"
+)
+
+const (
+	/*
+		- The SQL queries do a case-insensitive search.
+		- English words are normalized during search which means words like qualifying === qualify
+		- To disable the above normalization change "to_tsquery('english',$1)" to "to_tsquery($1)"
+		- Create GIN indexes : https://www.postgresql.org/docs/9.5/static/textsearch-tables.html#TEXTSEARCH-TABLES-INDEX
+
+	*/
+
+	// This SQL query is used when search is performed across workitem fields and workitem ID
+	testText = `select * from work_items WHERE
+		setweight(to_tsvector('english', id::text), 'A') ||
+		setweight(to_tsvector('english', fields::text), 'B') @@ to_tsquery('english',$1) and deleted_at is NULL`
```

Comment:
Fixed in : 
https://github.com/almighty/almighty-core/pull/395/commits/7a9501aa85f8b8e1c5b573349ecf8065eace875b